corona-warn-app/cwa-app-android
# FIX: Export test results shown in contact journal (EXPOSUREAPP-11441)

### Testing
- Scan some positive/negative PCR/RAT tests 
- Check tests in the contact journal
- Export Entries
- Tests should be visible in the exported data

### Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11441

### Contact Journal (Export of Data) Spec
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/main/docs/spec/contact-diary.md#export-of-data
